jenkins/pipeline.groovy

```diff
@@ -213,7 +217,7 @@ pipeline {
                                 fi
                                 rm -f ${WORKSPACE}/VERSION-${BUILD_NUMBER}
                             '''
-                            git branch: '8.0', url: 'https://github.com/Percona-Lab/ps-build'
+                            git branch: 'PS-8711', url: 'https://github.com/Percona-QA/ps-build'
```

Comment:
Is change of repo and fixed branch intended or were added for tests and should be reverted?

Reply:
thanks for the query yes, only for testing purpose -- before merge needs revert.